x/distribution/keeper/msg_server.go

```diff
@@ -166,3 +166,37 @@ func (k msgServer) UpdateParams(goCtx context.Context, req *types.MsgUpdateParam
 
 	return &types.MsgUpdateParamsResponse{}, nil
 }
+
+func (k msgServer) CommunityPoolSpend(goCtx context.Context, req *types.MsgCommunityPoolSpend) (*types.MsgCommunityPoolSpendResponse, error) {
```

Comment:
we should bump consensus version here (if it wasn't done already since 0.46)

Reply:
You need to bump consensus versions even when no state changed, correct?

Reply:
Yes when a module introduces a consensus-breaking change

Reply:
It's 2 on v0.46 and thus should be `3` here. Not sure why it got changed to 4.